tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -121,6 +121,7 @@ function deployBackend() {
 
     sed "s/SERVICE_NAME/${BACKEND_SERVICE_NAME}/g" app_template.yaml > app.yaml
     gcloud app deploy --quiet
+    sleep 1m
```

Comment:
print something + timestamp about this 1 minute sleep.  it will be easier to debug